src/network/netdev/netdev.c

```diff
@@ -995,15 +995,12 @@ int netdev_load_one(Manager *manager, const char *filename) {
         return 0;
 }
 
-int netdev_load(Manager *manager, bool reload) {
+int netdev_load(Manager *manager) {
         _cleanup_strv_free_ char **files = NULL;
         int r;
 
         assert(manager);
 
-        if (!reload)
-                hashmap_clear_with_destructor(manager->netdevs, netdev_unref);
```

Comment:
maybe add an `assert(hashmap_isempty(…` instead?

Reply:
No, when this is called on reload, the hashmap is _not_ empty.